### ISSUE TITLE
Support Gmail's smaller: and larger: size restrictions in CLI

### ISSUE THREAD

**Issue by caio1982** [caio1982/Lost-Photos-Found#20]

**Comment by cmatsuoka** [comment]
The easy way to limit size (or to filter using any other Gmail search operator) would be to use user-defined search criteria, such as in -S 'larger:10M'. Do you think a special CLI switch is necessary?

**Comment by caio1982** [comment]
Not really, I was just moving items from TODO to actual issues. Documenting useful criteria in the CLI as suggestions should be enough.
